### DISCUSSION TITLE
ByteArrayToImageSourceConverter bug?

### DISCUSSION THREAD

**Discussion by plettb** [CommunityToolkit/Maui#387] [Q&A] [answered]
Before I write this up as a bug I'm curious if anyone else is experiencing this, or if I'm doing something wrong.  I'm storing images in a database and using "ByteArrayToImageSourceConverter" to convert from a byte array to an image.  This works perfectly in the UWP version of my app, but doesn't appear to be working properly in the Android version.  I get no errors (binding or otherwise) - I just don't get an image in my app.

I searched the bugs and discussions but found nothing.  Any ideas?

**Comment by VladislavAntonyuk** [comment, marked as answer]
It's known .NET MAUI issue.

**Reply by TheCodeTraveler** [reply]
Yup! Here's the link to the bug: https://github.com/dotnet/maui/issues/6116

**Reply by plettb** [reply]
OK, I guess it was closed, but not in time for 17.2.0 Preview 5.0.  I'll try to be patient. :)

**Reply by TheCodeTraveler** [reply]
Yup! It should be in RC3

> this didn't make it for RC2, it should be fixed with RC3

(same bug, different/duplicate Issue):
https://github.com/dotnet/maui/issues/6067#issuecomment-1112229612

**Comment by CliffAgius** [comment]
Closed as answered.
